pkg/plugins/platform/local_test.go

```diff
@@ -62,13 +68,56 @@ func TestMain(m *testing.M) {
 		if err != nil {
 			log.Fatalf("Prepare() failed on TestSite.Prepare(), err=%v", err)
 		}
+
+		switchDir := TestSites[i].Chdir()
+		runTime := testcommon.TimeTrack(time.Now(), fmt.Sprintf("%s Start", TestSites[i].Name))
+
+		testcommon.ClearDockerEnv()
+
+		app, err := platform.GetPluginApp("local")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = app.Init(TestSites[i].Dir)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = app.Start()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		runTime()
+		switchDir()
 	}
 
 	log.Debugln("Running tests.")
 	testRun := m.Run()
 
-	for i := range TestSites {
-		TestSites[i].Cleanup()
+	for _, site := range TestSites {
+		runTime := testcommon.TimeTrack(time.Now(), fmt.Sprintf("%s Remove", site.Name))
+
+		testcommon.ClearDockerEnv()
+
+		app, err := platform.GetPluginApp("local")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = app.Init(site.Dir)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = app.Down(true)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		runTime()
+		site.Cleanup()
```

Comment:
It sure is a huge win doing the container stops before the site cleanups! Thanks!
I'm not entirely sure why we don't consolidate the app-down functionality into the site.Cleanup(), but this works for me.